index.js

```diff
@@ -95,19 +164,11 @@ getMove = function (control) {
 };
 
 module.exports = defineProperties(getFn(), {
-	width: d.gs(process.stdout.getWindowSize ? function () {
-		return process.stdout.getWindowSize()[0];
-	} : function () {
-		return tty.getWindowSize ? tty.getWindowSize()[1] : 0;
-	}),
-	height: d.gs(getHeight = process.stdout.getWindowSize ? function () {
-		return process.stdout.getWindowSize()[1];
-	} : function () {
-		return tty.getWindowSize ? tty.getWindowSize()[0] : 0;
-	}),
-	reset: d.gs(function () {
-		return repeat.call('\n', getHeight() - 1) + '\x1bc';
-	}),
+	trim: d(trim),
+	throbber: d(throbber),
+	width: d.gs(function () { return process.stdout.columns || 0; }),
+	height: d.gs(function () { return process.stdout.rows || 0; }),
+	reset: d.gs(function () { return '\n\x1bc'; }),
```

Comment:
`process.stdout` have `columns` and `keys` that will help with that, alone. Reset need just one line break. But I really only tested on Windows and Cygwin (that is not perfect). On next weekend I'll install Linux here to make tests in both envs.